### ISSUE TITLE
sample-chat-swift: App crashes on Logout, prints `Push notifications unsubscribe failed` and crash on logoutGroup.leave() with `EXC_BAD_INSTRUCTION` error

### ISSUE THREAD

**Issue by nyan2x** [QuickBlox/quickblox-ios-sdk#818]
**Environment details**
*iOS version 10.3*

**Did this work before?**
I cloned the repo yesterday June 9 and had been experiencing such error since.

**Expected behavior**
Should be able to logout successfully to test the different user. If it really received the message.

**Actual behavior**
Application crash on logout

**Logs**
```
Push failed to register with error: %@ Error Domain=NSCocoaErrorDomain Code=3010 "remote notifications are not supported in the simulator" UserInfo={NSLocalizedDescription=remote notifications are not supported in the simulator}
Push notifications unsubscribe failed
Push notifications unsubscribe failed
```

**Steps to reproduce the behavior**
1. Log in to any user
2. Logout

**Comment by VitaliyGuru** [comment]
@nyan2x, Confirmed this bug. Will be fixed in the next QB iOS SDK release.

**Comment by Pro100Andrey** [comment]
@nyan2x, fixed in 2.10
